rclcpp/test/test_node_options.cpp

```diff
@@ -35,14 +35,16 @@ TEST(TestNodeOptions, ros_args_only) {
   ASSERT_EQ(0, rcl_arguments_get_count_unparsed_ros(&rcl_options->arguments));
 
   char * node_name = nullptr;
-  EXPECT_EQ(RCL_RET_OK, rcl_remap_node_name(
+  EXPECT_EQ(
+    RCL_RET_OK, rcl_remap_node_name(
```

Comment:
This one is odd to me too. I understand why it did this one, but it would be better if there was a "don't wrap all if only the last (or only) argument is multiline", but I guess it probably doesn't have that.